README.md

```diff
@@ -5,7 +5,25 @@
 
 # Philanthropy Data Commons Service
 
-This project is related to [Philanthropy Data Commons](https://philanthropydatacommons.org) (PDC). Please visit https://philanthropydatacommons.org for an overview of PDC.
+This is the data-handling service layer for the [Philanthropy Data Commons](https://philanthropydatacommons.org) (PDC).
+
+The PDC is an access-controlled environment in which changemakers and funders can share funding proposals, both for improved efficiency (e.g., offering a "common grant application" in specific domains) and for opening up new possibilities in partnering and alliance-building.  The PDC is designed to enable cross-organizational data sharing while allowing organizations to maintain their own systems, practices, and data standards.
```

Comment:
I know the target audience of this is the existing PDC community -- wondering if we should define "changemakers" somewhere for those who might not be in that loop? (e.g. we could create a glossary of terms in `/docs/GLOSSARY.md` and link to it?)

Reply:
@slifty I like the idea of a glossary -- I don't think this PR needs to block on it, but it's a good idea.  The immediate audience is PDC folks, you are right, and they'll know the terms I'm using.  (The choice of "proposals" instead of "applications" was also deliberate, so that there would be no confusion between an application form and the submitted response to that form.  I realize that within the PDC code base "application" is the official term.)